rules/not-stale.ts

```diff
@@ -0,0 +1,31 @@
+import { danger } from "danger";
+import { IssueComment } from "github-webhook-event-types";
+
+export const STALE_LABEL = `stale?`;
+
+export const notStale = async () => {
+  const gh = (danger.github as any) as IssueComment;
+  const repo = gh.repository;
+  const labels = gh.issue.labels.map(i => i.name);
+  const opts = {
+    owner: repo.owner.login,
+    repo: repo.name,
+    number: gh.issue.number,
+    name: encodeURIComponent(STALE_LABEL)
+  } as any;
```

Comment:
Typescript is infuriating sometimes... 🙃

Reply:
This is the first time I've used it! So far, so good :)